pylintrc

```diff
@@ -166,6 +166,7 @@ disable=print-statement,
         abstract-method,
         arguments-differ,
         cyclic-import,
+        invalid-str-returned,
```

Comment:
This seems important, why is this disabled?

Reply:
Because it seems to give wrong results. It was raised on for example `__str__` method in models when this method exactly returns string